SUPPLIMENTARY-INFO.md

```diff
@@ -0,0 +1,42 @@
+# Supplimentary Information
+
+## Front End Folder Structure
```

Comment:
I'm not sure how useful this is. Documentation could get out of date as the project changes.